bundles/org.openhab.binding.deconz/README.md

```diff
@@ -39,11 +39,13 @@ These configuration parameters are available:
 
 | Parameter | Description                                                   | Type      | Default   |
 | :-------- | :------------------------------------------------------------ | :-------: | :-------: |
-| host      | Host address (hostname/ip:port) of deCONZ interface           | string    | n/a       |
+| host      | Host address (hostname/ip) of deCONZ interface                | string    | n/a       |
+| port      | Port of deCONZ interface (default: 80)                        | string    | n/a       |
```

Comment:
Should the 'default: 80' be in the 'Default' column here?

Also, it seems the config of deconz allows one to specify both port and wsPort for the deconz service. (see for example https://github.com/dresden-elektronik/phoscon-app-beta/issues/26 )
Is that something that should be considered in this change? As a separate configuration parameter?

Reply:
Nice catch. Thank for the hint.

Yes, it definitely makes sense to add ports for both: HTTP REST API and Websocket. I will look into it in the next couple of days.

Reply:
Done.